nixos/modules/services/backup/znapzend.nix

```diff
@@ -381,12 +399,22 @@ in
         '';
 
         serviceConfig = {
+          # znapzendzetup --import apparently tries to connect to the backup
+          # host 3 times with a timeout of 30 seconds, leading to a startup
+          # delay of >90s when the host is down, which is just above the default
+          # service timeout of 90 seconds. Increase the timeout so it doesn't
+          # make the service fail in that case.
+          TimeoutStartSec = 180;
+          # Needs to have write access to ZFS
```

Comment:
That doesn't necessarily require root (https://github.com/oetiker/znapzend/blob/master/README.md#running-by-an-unprivileged-user)